docker/Dockerfile.nvidia

```diff
@@ -16,41 +16,52 @@
 # 
 #   docker build --network=host --build-arg MPI_VER=4 --file docker/Dockerfile.nvidia --tag devito:nvidia .
 #
+# SQUASH:
+#   If you want a smaller image or to collapse/combine the smaller layers, try squashing your docker image
+#
+#   pip install docker-squash
+#   docker tag devito:nvidia devito:nvidia-pre
+#   docker-squash  -t devito:nvidia-pre devito:nvidia
+#
 # RUN: 
 #   docker run --gpus all --rm -it -p 8888:8888 -p 8787:8787 -p 8786:8786 devito:nvidia
-#   docker run --gpus all --rm -it -p 8888:8888 -p 8787:8787 -p 8786:8786 devito:nvidia python examples/seismic/acoustic/acoustic_example.py
+#   docker run --gpus all --rm -it devito:nvidia python examples/seismic/acoustic/acoustic_example.py
 #
 #   docker run --gpus all --rm -it -p 8888:8888 -p 8787:8787 -p 8786:8786 --device=/dev/infiniband/uverbs0 --device=/dev/infiniband/rdma_cm  devito:nvidia 
 #
 # to run in user context on a cluster with shared filesystem, you can add the correct user config as docker options e.g.:
 #   docker run --gpus all --rm -it -v `pwd`:`pwd` -w `pwd` -u $(id -u):$(id -g) devito:nvidia python examples/seismic/acoustic/acoustic_example.py
 #
 ##############################################################
-FROM python:3.8
+#FROM python:3.9 
```

Comment:
Are the comments to be removed?

Reply:
I was just try to illustrate that there was an alternate non-slim version of the base container. `#FROM python:3.9 ` could be removed, yes.

Reply:
I think, the slimmer, the better right?

Reply:
That is the ask from some of our the users of the container.